var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -57,6 +58,7 @@ class Emacs(AutotoolsPackage):
     depends_on('giflib', when='+X')
     depends_on('libx11', when='+X')
     depends_on('libxaw', when='+X toolkit=athena')
+    depends_on('gnutls', when='@26.1:')
```

Comment:
I think if you remove this line and explicitly enable/disable the configure arg in `configure_args` this PR should be good to go.